clickhouse/client.go

```diff
@@ -183,12 +204,31 @@ func (c *Client) GetService(serviceId string) (*Service, error) {
 		return nil, err
 	}
 
-	return &serviceResponse.Result, nil
+	service := serviceResponse.Result
+
+	req, err = http.NewRequest("GET", c.getServicePath(serviceId, "/privateEndpointConfig"), nil)
```

Comment:
proposed new schema [here](https://github.com/ClickHouse/control-plane/issues/6235)

we should use new regional endpoint to retrieve this data.